web/frontend/src/components/slam-2d-render.vue

```diff
@@ -350,12 +282,16 @@ const updatePointCloud = (pointcloud: Uint8Array) => {
     points,
     intersectionPlane,
     axesPos,
-    axesNeg
```

Comment:
[q]: why no more axesNeg

Reply:
It's still there, it's just marking this as a change because of the new trailing `,` :)